41_snapshots-btrfs

```diff
@@ -183,7 +189,7 @@ make_menu_entries()
         fi
         echo 'Loading Snapshot: "${snap_date_time}" "${snap_dir_name}"'
         echo 'Loading Kernel: "${k}" ...'
-        linux \"${boot_dir_root_grub}/"${k}"\" root="${LINUX_ROOT_DEVICE}" rw ${kernel_parameters} rootflags=subvol=\""${snap_dir_name}"\""
+        linux \"${boot_dir_root_grub}/"${k}"\" root="${LINUX_ROOT_DEVICE}" rw ${kernel_parameters} ${rootflags}subvol=\""${snap_dir_name}"\""
```

Comment:
What do you think about removing "rw" option ?
The option is set too early so it will never be used, as it is often included in `kernel_parameters`, or now, thanks to your work, in custom rootflags.

Reply:
Agreed.